app/main.py

```diff
@@ -1,8 +1,33 @@
 class Person:
-    # write your code here
-    pass
 
+    people: dict[str] = {}
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
+
+    def add_wife(self, wife: str) -> None:
+        if wife is not None:
```

Comment:
due to your type annotations it does not possible 
update typing: wife: Optional[str]

The same with husband